projectgenerator/gui/ili2db_options.py

```diff
@@ -18,6 +18,14 @@
  ***************************************************************************/
 """
 
+from projectgenerator.utils.qt_utils import (
+    make_file_selector,
+    make_save_file_selector,
+    Validators,
+    FileValidator,
+    NonEmptyStringValidator,
+    OverrideCursor
```

Comment:
That's a lot of imports there, of which I think some are not used. (e.g. can't find `OverrideCursor`)
PyCharm will list them in gray

Reply:
Yes. I don't use `make_save_file_selector` and `OverrideCursor` - I forgot to remove them after copy this part...